lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb

```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- Tthi is the buttons group area--> 
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=refactor_keys(product)%>" class="<%= refactor_class %>"><%= first(product) %></a>
```

Comment:
Let's do a first step by using what we have in plain air and don't worry if it looks to verbose - we'll have time to improve/refactor this bits of code later on